packages/serde_json/serde_json.0.0.2/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+synopsis: "JSON format support for Serde"
+maintainer: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+authors: ["Leandro Ostera <leandro@abstractmachines.dev>"]
+license: "MIT"
+homepage: "https://github.com/serde-ml/serde"
+bug-reports: "https://github.com/serde-ml/serde/issues"
+depends: [
+  "ocaml" {>= "5.1.1"}
+  "ppx_inline_test" {>= "v0.16.0"}
+  "qcheck" {with-test}
+  "rio" {>= "0.0.8"}
+  "serde" {= version}
+  "serde_derive" {with-test & = version}
+  "spices" {with-test}
```

Comment:
```suggestion
```

Reply:
I would keep
```
"spices" {with-test}
```
in case people want to run the tests locally and use `opam install --with-tests` for the dependencies